ios/harmonyProject.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
oh ok, i understand what this file is now. This file is related to your workspace settings, so it's safe to ignore this file as it's specific to your XCode IDE. We can safely add `ios/harmonyProject.xcworkspace/` to the `.gitignore`.